docs/maintainers/concept-exercise-style-guide.md

```diff
@@ -0,0 +1,56 @@
+# Formatting and Style Guide for Concept Exercise Files
+
+This document records the current styleguide applicable to Concept exercise Markdown files, which refer to the files contained in `.docs` and `.meta`. Please follow them when implementing a Concept Exercise; maintainers should also reference the Style Guide when reviewing pull requests. When the relevant language also includes a style guide, those preferences will take precedence over the ones here.
+
+## Headers
+
+ - Use level 2 headers, e.g. `##`
+ - Do not use top-level headers, e.g. `#`
+ - Do not use leading whitespace in the file.
+
+
+## Empty Documents
+
+If a document is empty, i.e. there is no `after.md` content, then do not check one in.
+
+
+## Links
+
+Exercism prefers "indirect reference" links. They are defined at the bottom of the Markdown file and mapped to a reference slug, and referenced by that slug in the text.
+
+This makes maintenance easier, since the link must only be updated once.
+
+Example:
+
+```
+I have a paragraph of text that links to the same page twice. 
+
+The [first link][indirect-reference] in one sentence. 
+
+Then, another sentence with the [link repeated][indirect-reference].
+
+[indirect-reference]: https://example.com/link-to-page
+```
+
+Links should be inline where possible, i.e. as `<a href="">text</a>` tags instead of bare URLs. This is accomplished using the Markdown bracketed notation, as exemplified above.
```

Comment:
It took me a couple of reads to understand this bit, maybe a "do this (example) not this (example)"?

Reply:
I also found it a bit hard to understand. Could you explain what type of mistake you want to prevent people from making?

Reply:
Yeah, this isn't very clear. Will clean it up. I was primarily referring to not putting links behind text, i.e. leaving the whole URL in the text. An example would help a lot.

Reply:
Okay great!